apps/angular-mongodb/src/app/components/register/register.component.html

```diff
@@ -69,6 +69,8 @@
       </div>
     </form>
 
-    <div class="alert alert-success" *ngIf="isSuccessful">Your registration is successful!</div>
+    <div class="alert alert-success" id="alert" *ngIf="isSuccessful">
```

Comment:
rename alert to successful_message_registration

Reply:
done